manifests/c/Chatterino/Chatterino/2.4.4/Chatterino.Chatterino.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created with Komac v1.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Chatterino.Chatterino
+PackageVersion: 2.4.4
+InstallerLocale: en-US
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: Chatterino2/chatterino.exe
+  PortableCommandAlias: Chatterino2/chatterino.exe
+Scope: user
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2023-05-13
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://github.com/Chatterino/chatterino2/releases/download/v2.4.4/Chatterino.Installer.exe
```

I just want to note that the installer from Github releases is just a re-upload of the one listed on the website, hosted on digitalocean.

```suggestion
  InstallerUrl: https://chatterino.fra1.digitaloceanspaces.com/bin/2.4.4/Chatterino%20Installer.exe
```